tests/lib/rules/default-param-last.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @fileoverview Test file for default-param-last
+ * @author Chiawen Chen
+ */
+"use strict";
+
+const rule = require("../../../lib/rules/default-param-last");
+const { RuleTester } = require("../../../lib/rule-tester");
+
+const SHOULD_BE_LAST = "shouldBeLast";
+
+const ruleTester = new RuleTester({
+    parserOptions: { ecmaVersion: 8 }
+});
+
+const cannedError = {
+    messageId: SHOULD_BE_LAST,
+    type: "AssignmentPattern"
+};
+
+ruleTester.run("default-param-last", rule, {
+    valid: [
+        "function f(a) {}",
+        "function f(a = 5) {}",
+        "function f(a, b = 5) {}",
+        "function f(a, b = 5, c = 5) {}",
+        "function f(a, b = 5, ...c) {}"
+    ],
```

Comment:
A couple of `valid` tests with arrow functions and function expressions would be nice for coverage.

Also, it would be good to have a test which shows that the rule will not crash on zero params. And a test with two params, both without default values.